EuclidProtocol/euclid-core-contracts
# Feature/ci pipeline

# Ci pipeline
Testing and liniting:

- linting done.
- check formating using cargo fmt.
- check efficiency of code using cargo clippy.
- compile the code.
- Build Artifacts:
- compile the contracts.
- upload it to actions.
